site/src/App/routes/guides/testing-guide.tsx

```diff
@@ -60,19 +60,20 @@ const page: Page = {
       </Text>
 
       <LinkableHeading level="3">
-        Should I use Enyzme or React Testing Library?
+        Which testing library should I use?
       </LinkableHeading>
 
       <Text>
-        While{' '}
-        <TextLink href="https://enzymejs.github.io/enzyme/">Enzyme</TextLink>{' '}
-        works perfectly well, we recommend{' '}
+        We recommend using{' '}
         <TextLink href="https://testing-library.com/docs/react-testing-library/intro">
           React Testing Library
         </TextLink>
-        . In fact, we use it internally in the Braid codebase. We feel it
-        encourages you to write more effective unit tests by decoupling them
-        from implementation detail.
+        . In fact, we use it internally in the Braid codebase. Unlike other
+        testing libraries such as{' '}
+        <TextLink href="https://enzymejs.github.io/enzyme/">Enzyme</TextLink>,{' '}
+        React Testing Library makes it difficult to test implementation details,
+        encouraging you to write tests that more closely resemble how your
+        software is used.
```

Comment:
I think saying the library makes something difficult makes it sound bad, even though it's actually useful constraint. IMO the previous wording was better: `React Testing Library encourages you to write more effective unit tests by decoupling them from implementation detail`